client/src/components/new-request/send-to-examination.vue

```diff
@@ -13,20 +13,17 @@
         <NameInput :is-read-only="true" class="mb-n2"/>
       </v-row>
 
-      <v-row no-gutters justify="center">
+      <v-row no-gutters justify="center" class="text-center">
         <v-col cols="auto" class="h5">
           <v-icon class="action-icon" :class="actionConfig.class">{{ actionConfig.icon }}</v-icon>
-          &nbsp;
           <span :class="actionConfig.class">{{ actionConfig.text }}</span>
         </v-col>
-        <v-col cols="12" class="py-2">
-          <p class="text-center copy-normal">
-            This name cannot be analyzed here and must be sent to staff for review.
-          </p>
+        <v-col cols="12">
+          <p :class="actionConfig.class">{{ actionConfig.subText }}</p>
         </v-col>
       </v-row>
 
-      <v-row class="text-center justify-center mt-n1 mb-n2">
+      <v-row class="text-center justify-center mb-n2">
```

Comment:
For next time...

To simplify layout and spacing, in the dev window, Elements tab, I hover on various elements and watch that they don't have any extra whitespace (esp below them).

Basically my rule is that all elements should be "tight" and only have explicit whitespace added (eg, above it for spacing to the previous element). It's almost never necessary to have whitespace below an element.

In this case, I suspect the element above this has bottom whitespace, which is why you have to rein some back in with this margin style. So you have updated THIS row to fix a problem in the previous row.